students/wachulec_maria/lesson_06_dicts_tuples_sets_args_kwargs/countries_and_cities.py

```diff
@@ -0,0 +1,15 @@
+def make_dict():
+    countries = {}
+    countries_amount = int(input())
+    for i in range(countries_amount):
+        country_and_city = input().split()
```

Comment:
it's totally fine, but take a lookat at this nice python shortcut
`country, *cities = input().split()`